README.md

```diff
@@ -34,7 +34,7 @@ Docs](https://rdhpcs-common-docs.rdhpcs.noaa.gov/wiki/index.php/Anaconda).
 
 There are a few specifics related to the AVID installation:
 
- - We use version: 4.5.12
+ - We use version: 4.8.3
```

Comment:
I'd prefer to see a new section in the readme on how to manage the conda module on Gaea. We do use 4.5.12 on Jet/Hera and I want to preserve the directions here in case we need to do it in the future.

Reply:
Ok, updated it again. Please take a look. On the long run, we can hopefully move to lua modulefiles for the other systems, too. They understand lua off the bat, no extra work required (as opposed to gaea). This would be better in line with the general move to lua modulefiles for UFS components (e.g. hpc-stack, NCEPLIBS, ...).